wgsl/index.bs

```diff
@@ -1989,9 +1989,9 @@ of a variable in [=address spaces/workgroup=] space.
 
 
 <div class='syntax' noexport='true'>
-  <dfn for=syntax>array_type_decl</dfn> :
+  <dfn for=syntax>array_type</dfn> :
```

Comment:
@dj2 I consider its consistency rather with "texture_and_sampler_types". Should we make that "texture_and_sampler_type_specifiers" or do _type here?

Reply:
I'd be inclined towards `texture_and_sampler_type_specifiers` then they're all consistenly named with specifier. Curious what others think.

Reply:
@dj2 Agreed, more opinion would help as those live at the same level and whether to nominate _type(s) or _type_specifier(s) should be consistent!

Reply:
@dneto0 I agree, as long as specifier does not further go into types themselves, I think the length is acceptable since other languages use a longer word "annotation" for such concept (see Swift). Thanks.